<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
		background-color: black;
		border: 1px solid black;
		padding: 5px; 	
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>650</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>650</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>650</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>650</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>650</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
		background-color: yellow;
		border: 1px solid yellow;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titlel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
		background-color: yellow;
		border: 1px solid yellow;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>TecFlix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnnext">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>650</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnprev">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>650</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnnext_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>650</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>-&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnprev_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>650</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;-</string>
    </property>
   </widget>
   <widget class="QLabel" name="pagel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
		background-color: yellow;
		border: 1px solid yellow;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnfile">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>FIle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnselect">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineselect">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string>movie_metadata.csv</string>
    </property>
   </widget>
   <widget class="QLabel" name="lposter">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>621</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
		background-color: black;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../movie.jpg</pixmap>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="bmovie1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../MV5BMjAwNDgxNTI0M15BMl5BanBnXkFtZTgwNTY4MDI1NzM@._V1_SY1000_CR0,0,631,1000_AL_.jpg</normaloff>../MV5BMjAwNDgxNTI0M15BMl5BanBnXkFtZTgwNTY4MDI1NzM@._V1_SY1000_CR0,0,631,1000_AL_.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>121</width>
      <height>171</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bmovie2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../MV5BMjAxMzY3NjcxNF5BMl5BanBnXkFtZTcwNTI5OTM0Mw@@._V1_UX182_CR0,0,182,268_AL_.jpg</normaloff>../MV5BMjAxMzY3NjcxNF5BMl5BanBnXkFtZTcwNTI5OTM0Mw@@._V1_UX182_CR0,0,182,268_AL_.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>121</width>
      <height>171</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bmovie3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../MV5BMTczNDk4NTQ0OV5BMl5BanBnXkFtZTcwNDAxMDgxNw@@._V1_SY1000_CR0,0,674,1000_AL_.jpg</normaloff>../MV5BMTczNDk4NTQ0OV5BMl5BanBnXkFtZTcwNDAxMDgxNw@@._V1_SY1000_CR0,0,674,1000_AL_.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>121</width>
      <height>171</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bmovie4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../MV5BMTkyOTkwNDc1N15BMl5BanBnXkFtZTgwNzkyMzk3NjM@._V1_SY1000_CR0,0,674,1000_AL_.jpg</normaloff>../MV5BMTkyOTkwNDc1N15BMl5BanBnXkFtZTgwNzkyMzk3NjM@._V1_SY1000_CR0,0,674,1000_AL_.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>121</width>
      <height>171</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bmovie5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../MV5BMzQxNzQzOTQwM15BMl5BanBnXkFtZTgwMDQ2NTcwODM@._V1_SY1000_CR0,0,674,1000_AL_.jpg</normaloff>../MV5BMzQxNzQzOTQwM15BMl5BanBnXkFtZTgwMDQ2NTcwODM@._V1_SY1000_CR0,0,674,1000_AL_.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>121</width>
      <height>171</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bmovie6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../MV5BNDk3NTEwNjc0MV5BMl5BanBnXkFtZTgwNzYxNTMwMzI@._V1_.jpg</normaloff>../MV5BNDk3NTEwNjc0MV5BMl5BanBnXkFtZTgwNzYxNTMwMzI@._V1_.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>121</width>
      <height>171</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bmovie7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../MV5BOTdiNzJlOWUtNWMwNS00NmFlLWI0YTEtZmI3YjIzZWUyY2Y3XkEyXkFqcGdeQXVyNjU0OTQ0OTY@._V1_SY1000_CR0,0,666,1000_AL_.jpg</normaloff>../MV5BOTdiNzJlOWUtNWMwNS00NmFlLWI0YTEtZmI3YjIzZWUyY2Y3XkEyXkFqcGdeQXVyNjU0OTQ0OTY@._V1_SY1000_CR0,0,666,1000_AL_.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>121</width>
      <height>171</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bmovie8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../MV5BOTg4ZTNkZmUtMzNlZi00YmFjLTk1MmUtNWQwNTM0YjcyNTNkXkEyXkFqcGdeQXVyNjg2NjQwMDQ@._V1_SY1000_CR0,0,674,1000_AL_.jpg</normaloff>../MV5BOTg4ZTNkZmUtMzNlZi00YmFjLTk1MmUtNWQwNTM0YjcyNTNkXkEyXkFqcGdeQXVyNjg2NjQwMDQ@._V1_SY1000_CR0,0,674,1000_AL_.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>121</width>
      <height>171</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bmovie9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../MV5BZDVkZmI0YzAtNzdjYi00ZjhhLWE1ODEtMWMzMWMzNDA0NmQ4XkEyXkFqcGdeQXVyNzYzODM3Mzg@._V1_SY1000_CR0,0,666,1000_AL_.jpg</normaloff>../MV5BZDVkZmI0YzAtNzdjYi00ZjhhLWE1ODEtMWMzMWMzNDA0NmQ4XkEyXkFqcGdeQXVyNzYzODM3Mzg@._V1_SY1000_CR0,0,666,1000_AL_.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>121</width>
      <height>171</height>
     </size>
    </property>
   </widget>
   <zorder>lposter</zorder>
   <zorder>btn1</zorder>
   <zorder>btn2</zorder>
   <zorder>btn3</zorder>
   <zorder>btn4</zorder>
   <zorder>btn5</zorder>
   <zorder>label_10</zorder>
   <zorder>titlel</zorder>
   <zorder>btnnext</zorder>
   <zorder>btnprev</zorder>
   <zorder>btnnext_2</zorder>
   <zorder>btnprev_2</zorder>
   <zorder>pagel</zorder>
   <zorder>btnfile</zorder>
   <zorder>btnselect</zorder>
   <zorder>lineselect</zorder>
   <zorder>bmovie1</zorder>
   <zorder>bmovie2</zorder>
   <zorder>bmovie3</zorder>
   <zorder>bmovie4</zorder>
   <zorder>bmovie5</zorder>
   <zorder>bmovie6</zorder>
   <zorder>bmovie7</zorder>
   <zorder>bmovie8</zorder>
   <zorder>bmovie9</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
